.grenrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "dataSource": "prs",
+    "prefix": "",
+    "onlyMilestones": false,
+    "ignoreTagsWith": ["v0.32.0", "0.0.3"],
```

Comment:
Why are these ignored? Is that just for immediate expediency? We can always add a tag and then tell it to only work with tags with a "v" prefix if that helps.

Reply:
These tags were created but no release was published. Changelog for these tags will be broken

Reply:
and there is no chance to fix it because creating releases for old tags is not possible